src/modules/init.js

```diff
@@ -0,0 +1,144 @@
+import {join} from 'path'
+import chalk from 'chalk'
+import moment from 'moment'
+import log from '../logger'
+import inquirer from 'inquirer'
+import {mkdir, writeFile} from 'fs'
+import {Promise, promisify, mapSeries} from 'bluebird'
+
+const bbMkdir = promisify(mkdir)
+const bbWriteFile = promisify(writeFile)
+
+function promptService () {
+  return Promise.try(() =>
+    inquirer.prompt({
+      name: 'service',
+      message: 'Choose the VTEX service you will use',
+      type: 'list',
+      choices: ['render'],
+    })
+  )
+  .then(({service}) => service)
+}
+
+function promptName () {
+  const message = 'The app name should only contain number, letters, underscores and hyphen.'
+  return Promise.try(() =>
+    inquirer.prompt({
+      name: 'name',
+      message: 'What\'s your VTEX app name?',
+      validate: s => /^[A-Za-z0-9\-_]+$/.test(s) || message,
+      filter: s => s.trim(),
+    })
+  )
+  .then(({name}) => name)
+}
+
+function promptVendor () {
+  const message = 'The vendor should only contain number, letters, underscores and hyphen.'
+  return Promise.try(() =>
+    inquirer.prompt({
+      name: 'vendor',
+      message: 'What\'s your VTEX app vendor?',
+      validate: s => /^[A-Za-z0-9\-_]+$/.test(s) || message,
+      filter: s => s.trim(),
+    })
+  )
+  .then(({vendor}) => vendor)
+}
+
+function promptTitle () {
+  return Promise.try(() =>
+    inquirer.prompt({
+      name: 'vendor',
+      message: 'What\'s your VTEX app title?',
+      filter: s => s.trim(),
+    })
+  )
+  .then(({title}) => title)
+}
+
+function promptDescription () {
+  return Promise.try(() =>
+    inquirer.prompt({
+      name: 'vendor',
+      message: 'What\'s your VTEX app description?',
+      filter: s => s.trim(),
+    })
+  )
+  .then(({description}) => description)
+}
+
+function createManifest (name, vendor, title, description) {
+  const [year, ...monthAndDay] = moment().format('YYYY-MM-DD').split('-')
+  const manifest = {
+    name,
+    vendor,
+    version: '1.0.0',
+    title,
+    description,
+    mustUpdateAt: `${Number(year) + 1}-${monthAndDay.join('-')}`,
+    categories: [],
+    settingsSchema: {},
+    dependencies: {
+      'vtex.renderjs': '0.x',
+      'vtex.placeholder': '0.x',
+      'npm:react': '15.1.0',
+    },
+  }
+  return JSON.stringify(manifest, null, 2)
+}
+
+export default {
+  init: {
+    handler: function () {
+      log.info('Hello! I will help you generate basic files and folder for your app.')
+      log.info('Please login before procceding.\n')
+      return this.login.handler()
```

Comment:
@firstdoit You're right! I forgot about the whitelist :/
Maybe it didn't prompt me because I was already logged.
I will remove this to take advantage of what's already there :)
Thanks for the reminder!